spark/components/masthead/vanilla/masthead.js

```diff
@@ -1,7 +1,10 @@
 /* global document window */
+import { throttle } from 'lodash';
```

Comment:
hey @RVMendoza , are these changes ones to bring the masthead.js file up to date?

Reply:
yup!